src/components/MappingEvents/MappingEventToolbar.js

```diff
@@ -132,20 +140,29 @@ const MappingEventToolbar = ({
         <img className="mapping-event-image" src={imageSource} alt="" />
         <div className="mapping-event-description">{mappingEvent.description}</div>
         <section className="statistics" aria-label={statisticsRegionAriaLabel}>
-          <div>
+          <div className="statistic">
             <div className="statistics-count">
               <MapPinWithPlusIcon />
               <span>{mappingEvent.statistics.mappedPlacesCount}</span>
             </div>
             <div className="statistics-description">{mappedPlacesLabel}</div>
           </div>
-          <div>
+          <div className="statistic">
             <div className="statistics-count">
-              <BellIcon />
+              <UserIcon />
               <span>{mappingEvent.statistics.invitedParticipantCount}</span>
             </div>
             <div className="statistics-description">{inviteesCountAriaLabel}</div>
           </div>
+          {remainingDaysLeft && (
+            <div className="statistic">
```

Comment:
0: Wrap in own styled component? It's just a line of css now, so might not be worth it :)

Reply:
But it could include all the other statistics-rules.

Reply:
Also NITPICK: `statistics` vs `statistic`?

Reply:
Yeah, let me see if a separate component is sensible here.

`statistics` means the container and `statistic` each element within. What would you nit-correct? :)

Reply:
I had some incosistencies, just realized. The container is now in plural form and the class names for each individitual "statistic" are in singular form now.
Maybe that's what you meant? :)

Reply:
I also put the statistics stuff into its own subcomponent. Is a bit more clean this way.